### ISSUE TITLE
The training code of concept classification

### ISSUE THREAD

**Issue by ShiYaya** [jacobswan1/ViTCAP#1]
Can you provide the training code of concept classification?

**Comment by danielgf0824** [comment]
+1

**Comment by jacobswan1** [comment]
Hi there, I did not maintain the code for concept classification but this part is simple: 

Use the CLS token embedding after decoder and link it with a classification layer with N numbers of categories. For classification loss use src/layers/loss/FocalLossWithLogitsNegLoss. 

Please let me know if you have more doubt on this one.

**Comment by TreezzZ** [comment]
> Hi there, I did not maintain the code for concept classification but this part is simple:
> 
> Use the CLS token embedding after decoder and link it with a classification layer with N numbers of categories. For classification loss use src/layers/loss/FocalLossWithLogitsNegLoss.
> 
> Please let me know if you have more doubt on this one.

Thanks for your work! I would like to know what is the accuracy of multi-label classification in CTN. I try to adopt CTN in my task, but the loss isn't decreased and the accuracy is only 12%.
